ROADMAP.md

```diff
@@ -0,0 +1,34 @@
+# Roadmap
+
+## Phase 1 (v0.1.0):
+
+- [ ] Create a basic controller that supports PID control.
```

Comment:
These are goals at a high level view right? For this one for example I imagine that defining and implementing the behavior tree is implicit in the goal.

Reply:
Some of them are high level, but I really forgot to add the behavior tree task in the roadmap. Good catch!